### ISSUE TITLE
is there a way to fetch history borrow/lending rate without actually borrowing/lending the particular coin?

### ISSUE THREAD

**Issue by ksl871** [ftexchange/ftx#46]
ex. I want to get USD history borrow/lending rates all the way back to 2019. Do we have APIs that support this scenario? (similar to getting history funding rate for BTC-PERP)

**Comment by jchoffmann** [comment]
I was wondering the same thing. Did you find a way?

**Comment by ksl871** [comment]
no, but I asked in their Telegram (https://t.me/ftx_api_support), they said it's in their backlog

**Comment by tong-zipmex** [comment]
are you done with it? 
Im looking for the answer same question here.

Please help
